website/sidebars.js

```diff
@@ -1,31 +1,8 @@
-/**
- * Creating a sidebar enables you to:
- - create an ordered group of docs
- - render a sidebar for each doc of that group
- - provide next/previous navigation
-
- The sidebars can be generated from the filesystem, or explicitly defined here.
-
- Create as many sidebars as you want.
- */
-
-// @ts-check
-
-/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
-const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
+module.exports = {
+  mySidebar: [
```

Comment:
it looks like all these changes on this file are like a no-op change 
You also removed the TypeScript types / checks

Reply:
It sticks to Docusaurus documentation.

Reply:
the types should not be removed

```
// @ts-check

/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */

```

also from what I see you didn't introduced any changes in this file except removing all the comments

Reply:
The code looked to me as boilerplate from an older version.
I took the code from the current docs, for clarity: it's exactly the same as in the docs, so there is no reason to search to understand the differences.

Reply:
the code is generated from the docusaurus generator and it includes the Types so we have completion with IDE
here you removed all type checking entries

Reply:
// @ts-check

/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */

are back, what else is missing?

Reply:
they need to be where the field/type is used

Reply:
Last commit should be fine.